patchwork-events-world/src/main/java/com/patchworkmc/mixin/event/world/MixinClientWorld.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package com.patchworkmc.mixin.event.world;
+
+import java.util.function.BiFunction;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.WorldEvent;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.util.profiler.Profiler;
+import net.minecraft.world.chunk.ChunkManager;
+import net.minecraft.world.dimension.Dimension;
+import net.minecraft.world.dimension.DimensionType;
+import net.minecraft.world.level.LevelProperties;
+import net.minecraft.world.World;
+import net.minecraft.client.world.ClientWorld;
+
+@Mixin(ClientWorld.class)
+public abstract class MixinClientWorld extends World {
+	protected MixinClientWorld(LevelProperties levelProperties, DimensionType dimensionType, BiFunction<World, Dimension, ChunkManager> chunkManagerProvider, Profiler profiler, boolean isClient) {
+		super(levelProperties, dimensionType, chunkManagerProvider, profiler, isClient);
+	}
+
+	@Inject(method = "<init>", at = @At(value = "TAIL"))
+	private void hookConstructor(CallbackInfo info) {
```

Comment:
This may be more descriptive.
```suggestion
	private void postConstruct(CallbackInfo info) {
```